src/specBuilder/signal/signalSpecBuilder.ts

```diff
@@ -94,3 +97,95 @@ export const getLegendLabelsSeriesSignal = (value: unknown = null): Signal => {
 export const getGenericSignal = (name: string, value: unknown = null): Signal => {
 	return { name, value };
 };
+
+//TODO: Add documentation
+export const getAnimationSignals = (name: string): Signal[] => {
+	return [
+		getRSCAnimation(),
```

Comment:
We should use `Rsc` in all these function names.  Using an acronym is an edge case for CamelCase and the format can go either way.  But, because the project already uses `RscChart` future developers get a lot of value from us keeping consistent to `Rsc`